### ISSUE TITLE
Pixi v5

### ISSUE THREAD

**Issue by jimmyff** [emergent-design/pixi.dart#10]
Hey just wondering if you're planning to move to Pixi v5?

Thanks

**Comment by parnham** [comment]
Yes there are plans to move to Pixi v5 but unfortunately a lack of time has impeded progress. 
We use this library for a product we're developing and since Pixi v4 is stable this project has been left alone for a bit, for which I apologise. 
Are there specific features of v5 that you require?

**Comment by onuse** [comment]
Speaking for myself, I think it generally comes down to that I want to use Pixi in my Dart-project and it is difficult to get a good of grasp what has changed between 4 and 5. Which means it's hard to tell what, if anything or alot of things, needs to be rewritten/redesigned when this library gets updated.
If there is a fundamental difference, it will probably affect how I use this library. I just dont know.

**Comment by franciscoleal** [comment]
Hi, can I use this package in version 2.0 of the flutter? Do you have a forecast for when updates will be released?

**Comment by parnham** [comment]
This is a wrapper around the pixi.js library and is therefore a browser technology and not compatible with flutter. I've not used flutter myself and so do not know if it's possible to export a web application that can use interop libraries.

We're still using this within a commercial application based on angular and since v4 is stable and contains everything we need there has been no time available to migrate to later versions of the underlying pixi js library. 
We have been looking at updating it for the newer version of the dart SDK, but not sure at this stage whether we will stick with v4 of pixi.
